.github/workflows/test.yml

```diff
@@ -447,10 +448,12 @@ jobs:
         fi
 
         # tutorial has some performance measurements that can be flaky if we run them in parallel
-        TEST_GROUPS_SERIAL="performance tutorial"
+        TEST_GROUPS_SERIAL="tutorial"
```

Comment:
There's one performance test in the tutorials I think. How about changing it to not be an error in this PR?

Reply:
also at least one in `auto_schedule` (see https://github.com/halide/Halide/issues/4749). Not sure what to do about that one.